app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Header.scss';
+
+type HeaderProps = {
+    logoUrl: string;
```

Comment:
optional: you can safely hardcode the logo URL in this component. This kind of stuff doesn't change per component's instantiation.